### DISCUSSION TITLE
[prefect 1] Cluster failed to start: Timed out trying to connect

### DISCUSSION THREAD

**Discussion by shreyaslr** [PrefectHQ/prefect#16033] [General]
### Bug summary

Describe the issue:

I am using prefect and dask getting below error

`RuntimeError: Cluster failed to start: Timed out trying to connect to tcp://dask-root-d.default:8786 after 30 s`

even though the environment value is set to 300 seconds

`ENV DASK_DISTRIBUTED__COMM__TIMEOUTS__CONNECT=300s`

Environment:

dask-kubernetes==2023.7.0
dask==2023.7.0
distributed==2023.7.0
prefect==1.2.4

### Version info

```Text
prefect==1.2.4
```

**Comment by zzstoatzz** [comment]
hi @shreyaslr - a couple things:

- `prefect 1.x` is not actively maintained
- the problem appears to be about connectivity to your own dask cluster, i.e its not clear how this problem relates to prefect
